src/deployment/cd.md

```diff
@@ -193,6 +193,111 @@ secrets in pull requests that you accept and merge.
          * `cd android` or `cd ios`
          * `bundle exec fastlane [name of the lane]`
 
+## Xcode Cloud
+
+[Xcode Cloud][] is a continuous integration and delivery service for building,
+testing, and distributing apps and frameworks for Apple platforms.
+
+### Requirements
+
+* Xcode 13.4.1 or higher.
+* Be enrolled in the [Apple Developer Program][].
+
+### Custom build script
+
+Xcode Cloud recognizes three different [custom build scripts][] that can be
+used to perform additional tasks at a designated time. It also includes a set
```

Comment:
The custom build scripts reference seems dangling now, maybe move it to where you first reference the script? Something like:

```
#### Post-clone script

Leverage the post-clone [custom build script][] that
runs after Xcode Cloud clones your Git repository using
the following instructions:
```
If you take this suggestion, don't forget to update the link text in the bottom references.